helm/alfresco-content-services/templates/config-imagemagick.yaml

```diff
@@ -12,10 +12,10 @@ data:
   {{ $key }}: {{ $val | quote }}
   {{- end }}
   {{- end }}
-  {{- if index .Values "alfresco-infrastructure" "activemq" "enabled" }}
+  {{- if index .Values "activemq" "enabled" }}
```

Comment:
we can go with the simpler .Values.activemq.enabled as we don't have the dash anymore

Reply:
Done